db/migrate/20120501184422_remove_nucs.rb

```diff
@@ -1,22 +0,0 @@
-class RemoveNucs < ActiveRecord::Migration
```

Comment:
The `nucs_` table creation migrations exist within the nucs engine in the NU branch. That's why I removed them from the initial schema and removed this migration.